README.md

```diff
@@ -30,45 +34,70 @@ log.warn('My Warning');
 log.error('My Error');
 log.critical('My Critical Error');
 ```
+
 ### Formatting
 
 Using util.format(...)
+
 - %s - String.
 - %d - Number (both integer and float).
 - %j - JSON. Replaced with the string '[Circular]' if the argument contains circular references.
 - %% - single percent sign ('%'). This does not consume an argument.
+
 ```javascript
-log.info('My Formatted %s', 'Message')
->> 'My Formatted Message'
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
+log.info('My Formatted %s', 'Message') >> 'My Formatted Message';
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
 ```
 
 Json Formatting
+
 ```javascript
-log.info({a: 'aloha'})
->> {a: 'aloha'}
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
+log.info({ a: 'aloha' }) >> { a: 'aloha' };
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
 ```
 
 ## Configuring a Provider
+
 ```javascript
 LoggerFactory.setFactory(<factory-function>);
 LoggerFacotry.setFactory(ConsoleLogger);
 ```
 
+### Log Levels
+
+When setting a factory provider, you can also set a level to ensure not to send logs if the level is too low.
+
+#### Import Level
+
+```javascript
+import { Level } from 'slf';
+```
+
```

Comment:
I think most of this can be removed. We don't need to document how to import the `Level`.

```suggestion
When setting a factory provider, you can also set a level to ensure not to send logs if the level is too low.
```